lib/ReactViews/Map/Panels/SharePanel/SharePanel.tsx

```diff
@@ -0,0 +1,667 @@
+"use strict";
+
+import classNames from "classnames";
+import { observer } from "mobx-react";
+import React, { ChangeEvent, MouseEventHandler } from "react";
+import { Trans, WithTranslation, withTranslation } from "react-i18next";
+import defined from "terriajs-cesium/Source/Core/defined";
+import Clipboard from "../../../Clipboard";
+import Icon from "../../../../Styled/Icon";
+import Loader from "../../../Loader";
+import MenuPanelAlias from "../../../StandardUserInterface/customizable/MenuPanel";
+import Input from "../../../../Styled/Input";
+import DropdownStyles from "../panel.scss";
+import {
+  buildShareLink,
+  buildShortShareLink,
+  canShorten,
+  isShareable
+} from "./BuildShareLink";
+import Styles from "./share-panel.scss";
+import StorySharePanelAlias from "./StorySharePanel";
+import {
+  Category,
+  ShareAction
+} from "../../../../Core/AnalyticEvents/analyticEvents";
+
+import { downloadImg } from "./Print/PrintView";
+import { IReactionDisposer, reaction } from "mobx";
+import Checkbox from "../../../../Styled/Checkbox";
+import Text from "../../../../Styled/Text";
+import Terria from "../../../../Models/Terria";
+import ViewState from "../../../../ReactViewModels/ViewState";
+import { TFunction } from "i18next";
+import CatalogMemberMixin from "../../../../ModelMixins/CatalogMemberMixin";
+
+// Handle incorrect type checking
+const MenuPanel = MenuPanelAlias as any;
+const StorySharePanel = StorySharePanelAlias as any;
```

Comment:
If you're importing old JSX files - I prefer to use `require` instead

```suggestion
const MenuPanel = require("../../../StandardUserInterface/customizable/MenuPanel")
  .default;
const StorySharePanel = require("StorySharePanel").default;
```

Reply:
OK applied